Document k6/experimental/redis module (#768)
grafana/k6-docs: src/data/markdown/docs/02 javascript api/13 k6-redis/20 Options.md

---
title: 'Options'
excerpt: 'Options allow to fine tune how a Redis client behaves and interacts with a Redis server or cluster.'
---

> The [Redis Client](/javascript-api/k6-redis/client) can be configured at construction time, using the [Options](#options) object. We recommend passing the options to the constructor as an argument, as shown in the example below, and pass the most common options such as the `addrs` and `password` to the constructor from the environment, as shown in the example below.

Reviewer: ```suggestion
You can configure the [Redis Client](/javascript-api/k6-redis/client) at construction time with the [Options](#options) object.
We recommend passing the options to the constructor as an argument,  then passing the most common options, such as the `addrs` and `password`, to the constructor from the environment.

The following snippet provides an example:
```
